lib/geoengineer/resources/aws_route53_record.rb

```diff
@@ -49,7 +49,7 @@ def self._fetch_zones(provider)
   end
 
   def self._fetch_records_for_zone(provider, zone)
-    zone_id = zone[:id].gsub(/^\/hostedzone\//, '')
+    zone_id = zone[:id].gsub(%r{^\/hostedzone\/}, '')
```

Comment:
I don't think you need to escape the forward slashes once you're using `%r`, though I guess it ends up with the same result:

```
irb(main):001:0> '/hosted_zone/' =~ %r{^/hosted_zone/}
=> 0
irb(main):002:0> '/hosted_zone/' =~ %r{^\/hosted_zone\/}
=> 0
```